### PR TITLE
pages: update getting started titles so they're more descriptive for users

### PR DESCRIPTION
**Pull request by admah**
Currently these are the options:
Screenshot 2024-01-12 at 4 48 06 PM

This PR makes the titles more descriptive to help users know more about the docs within.

C3      => create-cloudflare cli (C3)
Guide => Manual Setup

### CODE REVIEW

File: content/pages/get-started/guide.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Guide
+title: Manual setup
```
Comment (deadlypants1973) [inline content/pages/get-started/guide.md]:
```suggestion
title: Dashboard guide
```

**Review by deadlypants1973 — APPROVED** [review]
My proposal would be to say "Dashboard guide" to be accurate to the text. Manual could mean different things depending on perspective

**Comment by deadlypants1973** [comment]
I think this can be closed per my latest update?

**Comment by admah** [comment]
Closing as duplicate of #13076

**Pull request closed without merging**